modules/services/tailscale.nix

```diff
@@ -23,26 +22,32 @@ in
       description = lib.mdDoc "The package to use for tailscale";
     };
 
-    magicDNS = {
-      enable = mkOption {
-        type = types.bool;
-        default = false;
-        example = true;
-        description = lib.mdDoc "Whether to configure networking to work with Tailscale's MagicDNS.";
-      };
+    overrideLocalDns = mkOption {
```

Comment:
I think the general style across Nix is to uppercase acronyms like DNS (there are a few other examples just in this diff), but I’m not a stickler.